### ISSUE TITLE
Cannot run katoolin3.py

### ISSUE THREAD

**Issue by MiTeePingWin** [s-h-3-l-l/katoolin3#7]
Hello,
I'm running Samsung Dex with Ubuntu 16.04. I get the error when I try to run katoolin3:
dextop@localhost:/katoolin3$ sudo katoolin3
  File "/usr/local/bin/katoolin3", line 456
    return f"{index}) {self._options[index].text}"
                                                 ^
SyntaxError: invalid syntax
dextop@localhost:/katoolin3$

**Comment by MiTeePingWin** [comment]
I think I got further, python3.6 is required. I'm working on getting that running now. Stuck on another problem.

**Comment by MiTeePingWin** [comment]
I'm stuck, it says python-apt is required but is already installed. Tried uninstalling, force reinstalling, etc. No luck yet.

**Comment by xnetcat** [comment]
Maybe your using pip for python2
try `python3 -m pip install python-apt`

**Comment by MiTeePingWin** [comment]
When I run python3 -m pip install python-apt I get:
python3: can't open file 'pip': [Errno 2] No such file or directory
Same for python pip

If I run pip, it's recognized and gives me the command list. 
pip install python-apt downloads the package and says:
Requirement already satisfied: python-apt in /usr/lib/python2.7/dist-packages (1.1.0b1+ubuntu0.16.4.5)

Running: 'python-apt' returns: command not found

It looks like the package commands are just not being identified?

**Comment by s-h-3-l-l** [comment]
I'm gonna rewrite katoolin3 to make it backwards compatible with python3.5.
Please wait until the next update is available.

**Comment by s-h-3-l-l** [comment]
The update is ready.
Please update your repo clone with [update.sh](https://github.com/s-h-3-l-l/katoolin3/blob/master/update.sh) and tell me if works (make sure it runs with python3.5).

This was fixed in commit 5f3abfdcf2f971042d2c225abebba4a026ec1260.

**Comment by MiTeePingWin** [comment]
Almost a success. I deleted my imaged and started fresh with your new update.
Image came with python3.5. Needed to install python3-apt, no issues.
Needed to mkdir /usr/local/bin
Ran sudo katoolin3, it worked this time.

I tried to install Armitage, got the error, Summarized as:
```
Reading package lists...
Traceback, line 709
SystemError: pkgProblemResolver::Resolve generated breaks, this may be caused by held packages

AttributeError: module 'apt_pkg' has no attribute 'Error'
```

I need to apt-get upgrade, I'll do that in a few hours and I'll try again to see if that's the problem. Apologies for the abbreviated error code, I'm running this on a Galaxy S8 and can't copy/paste from it.
